libsolidity/grammar.txt

```diff
@@ -1,48 +1,96 @@
-ContractDefinition = ( 'contract' | 'library' ) Identifier
+ContractDefinition = ('import' Imports ';')*
```

Comment:
The import statement is not part of the contract. The initial nonterminal should be `SourceUnit`